src/fu-tool.c

```diff
@@ -1301,6 +1301,68 @@ fu_util_detach (FuUtilPrivate *priv, gchar **values, GError **error)
 	return fu_device_detach (device, error);
 }
 
+static gboolean
+fu_util_unbind_driver (FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(FuDevice) device = NULL;
+	g_autoptr(FuDeviceLocker) locker = NULL;
+
+	/* load engine */
+	if (!fu_util_start_engine (priv, FU_ENGINE_LOAD_FLAG_NONE, error))
+		return FALSE;
+
+	/* get device */
+	if (g_strv_length (values) == 1) {
+		device = fu_util_get_device (priv, values[0], error);
+		if (device == NULL)
+			return FALSE;
+	} else {
+		device = fu_util_prompt_for_device (priv, NULL, error);
+		if (device == NULL)
+			return FALSE;
+	}
```

Comment:
I think you can simplify this to have the `(device == NULL)` check pop out of the if loop since both cases end with it.

Reply:
Fixed.